cluster-autoscaler/core/utils.go

```diff
@@ -407,7 +407,7 @@ func removeOldUnregisteredNodes(unregisteredNodes []clusterstate.UnregisteredNod
 			}
 			size, err := nodeGroup.TargetSize()
 			if err != nil {
-				klog.Warningf("Failed to get node group size, err: %v", err)
+				klog.Warningf("Failed to get node group size for %s: %v", unregisteredNode.Node.Name, err)
```

Comment:
If already changing this message then add node group id to it also. So it is sth like
"Failed to get node group size; unregisteredNode=%v; nodeGroup=%v; err=%v"